### PR TITLE
Replace strcpy and strncpy by new inline helper function

### PR DESCRIPTION
**Pull request by stweil**

### CODE REVIEW

**Comment by egorpugin** [comment]
We should name it `copy_string_remove_after_transition_to_std_strings()`.

File: src/ccmain/ltrresultiterator.cpp [hunk 1]
```diff
@@ -310,11 +308,7 @@ char *LTRResultIterator::WordTruthUTF8Text() const {
   if (!HasTruthString()) {
```
Comment (egorpugin) [inline src/ccmain/ltrresultiterator.cpp:308]: What if we change this function return type to std::string?
```
char *LTRResultIterator::WordTruthUTF8Text() const {
```
to
```
std::string LTRResultIterator::WordTruthUTF8Text() const {
```
and also all calling sites?

**Comment by stweil** [comment]
> We should name it `copy_string__remove_after_transition_to_std_strings()`.

That's a little bit long, but a comment which basically says the same thing could be added in helpers.h.

**Review by egorpugin — APPROVED** [review]

File: src/ccutil/helpers.h [hunk 2]
```diff
@@ -35,6 +35,15 @@
 
 namespace tesseract {
 
+// Copy a std::string to a newly allocated char *.
```
Comment (stweil) [inline src/ccutil/helpers.h:38]:
```suggestion
// Copy a std::string to a newly allocated char *.
// TODO: Remove this function once the related code has been converted to use std::string.
```

**Pull request merged**